cmd/agent/subcommands/flare/command.go

```diff
@@ -76,6 +76,7 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 					SysprobeConfigParams: sysprobeconfig.NewParams(sysprobeconfig.WithSysProbeConfFilePath(globalParams.SysProbeConfFilePath)),
 					LogParams:            log.LogForOneShot("CORE", "off", false),
 				}),
+				flare.Module,
```

Comment:
Since `flare` was only used in two places, this seemed like the better approach, but I'm happy to change if you feel strongly. Having `flare` imported in `comp/core` means that anyone importing `comp/core` gets a lot of wanted dependencies (like `k8s.io` and such).

Reply:
It's true. The plan is to empty `pkg/flare` package when migrating to component. So once the migration is over, including the `flare` component will add 0 dependencies (since the flare component will use callback registered by the already imported dependencies).

Since migration will take some time we might want to move flare to it's dedicated bundle for now, maybe ?

Reply:
A bundle is not required for a single dependency, since you can directly use `flare.Module`, as I have. I'm not aware of any benefit of creating a bundle for a single dep.

Reply:
Indeed. We tried to have the logic of `bundle/comp` but we could have an exception.